tests/unit/handle-changes-test.js

```diff
@@ -0,0 +1,44 @@
+var simple = require('simple-mock')
+var test = require('tape').test
+
+var handleChanges = require('../../lib/helpers/handle-changes')
+
+test('handleChanges changes during bootstrapping', function (t) {
+  t.plan(2)
+
+  var change = {
+    doc: {
+      foo: 'bar'
+    },
+    id: 'id1234'
+  }
+  var changesSpy = {
+    on: simple.spy().callFn(function (changeEvent, callback) {
+      callback(change)
+    })
+  }
+
+  var state = {
+    db: {
+      allDocs: simple.spy().resolveWith({ rows: [] }),
+      changes: simple.spy().returnWith(changesSpy)
```

Comment:
I think that's what I like best about `simple-mock` -- they know that the dividing lines between mock, stubs, spies, etc is contentious and so they don't provide much of an opinion on the matter.  It just works! :smile: 

But you're right -- they should be mocks.